modules/@apostrophecms/modal/ui/apos/components/AposModalConfirm.vue

```diff
@@ -3,7 +3,7 @@
     :modal="modal"
     class="apos-confirm"
     :class="{ 'apos-confirm--tiny': options.tiny }"
-    v-on="mode !== 'alert' ? { esc: cancel } : null"
+    v-on="mode !== 'alert' ? { esc: close } : null"
```

Comment:
Because in the new modal I need three results possible:
* True
* False
* Null (when the user  clicks on the cross..)
Before we had only two steps. Since null is false it won't break any existing behavior.
So the new close method return null instead of false.